packages/storybook-builder-parcel/preset.js

```diff
@@ -119,10 +119,15 @@ async function createParcel(options, isDev = false) {
         resolveFrom: __filename,
       },
     ],
-    defaultTargetOptions: {
-      distDir: options.outputDir,
-      publicUrl: "./",
-      shouldScopeHoist: isDev ? false : true,
-    },
+    targets: {
```

Comment:
I do not recall why we made these changes. Devon Govett can hopefully shed some light here